whitelist.json

```diff
@@ -1 +1 @@
-["admin", "ugm"]
+["admin", "ugm", "unmesh", "haha"]
```

Comment:
This file will need to be made empty and created on first run so that it is unique to its own instance. Currently the contents are being carried over from our repl.it instance.